torch/nn/functional.py

```diff
@@ -1675,3 +1675,32 @@ def normalize(input, p=2, dim=1, eps=1e-12):
         eps (float): small value to avoid division by zero. Default: 1e-12
     """
     return input / input.norm(p, dim, True).clamp(min=eps).expand_as(input)
+
+
+def unfold(input, kernel_size, dilation=1, padding=0, stride=1):
```

Comment:
Could you add this to the docs and add a quick docstring pointing the viewer to nn.Unfold? Something like "See :class:`~torch.nn.Unfold` for details"

Reply:
Done